src/veh_interact.cpp

```diff
@@ -796,9 +810,15 @@ void veh_interact::do_install()
                 tab = ( tab < tab_list.size() - 1 ) ? tab + 1 : 0;
             }
 
-            copy_if(can_mount.begin(), can_mount.end(), back_inserter(tab_vparts), tab_filters[tab]);
-        }
-        else {
+            copy_if(can_mount.begin(), can_mount.end(), back_inserter(tab_vparts), part_filter);
+        } else if (action == "FILTER") {
+            sFilter = string_input_popup(_("Name filter:"), 55, sFilter,
+                    ("UP: history, CTRL-U clear line, ESC: abort, ENTER: save"), "veh_part_filter", 256);
```

Comment:
There's a `_` missing, it should be `_("UP: ... )"`, the `_` is important as it does the translation.